config/deploy/staging.rb

```diff
@@ -0,0 +1,8 @@
+role :app, %w{deploy@example.com}
```

Comment:
You know, on further though I think it would be safe to just remove this file.

My apologies for the backwards and forwards.